Text/PresidioPIIAnonymization/README.md

```diff
@@ -0,0 +1,215 @@
+# Text PII Anonymization Power Skill
+
+[Presidio](https://github.com/microsoft/presidio) is an open-source tool to recognize, analyze and anonymize personally identifiable information (PII). Using trained ML models, Presidio was built to ensure sensitive text is properly managed and governed.
+
+This Power Skill uses [Presidio](https://github.com/microsoft/presidio) Analyzer and Anonymizer
+to find and remove PII entities. Even though Presidio supports several anonymization methods (hash, encrypt, redact, replace, mask), 
+this Power Skill only uses redact and removes the PIIs completely from the text  .
+
+This skill is ideal for finding and removing PII entities from the search text.
+
+## Requirements
+
+In addition to the common requirements described in the root [README.md](https://github.com/Azure-Samples/azure-search-power-skills/blob/master/README.md) 
+file, this Power Skill requires spacy en_core_web_lg module being downloaded:
+```python
+python -m spacy download en_core_web_lg
+```
+
+To run this PowerSkill you will need:
+* docker
+* An Azure Blob storage container
+* A provisioned Azure Cognitive Search (ACS) instance 
+* A provisioned Azure Container Registry
+* A Cognitive Services key in the region you deploy ACS to
+
+Below is a full working example that you can get working E2E on sample data.
+
+## How to implement
+
+This section describes how to get this working on sample data and how it can be amended for your data.
+ 
+1) ### Data
+   The first step is to view the sample data files here [train data](data/). 
+1) ### Run the API
+   The next step is to run the API locally and test the model against a test record. Create a local python environment
+   and install the requirements:
+   
+   ```python
+   python -m pip install -r requirements.txt
+   ```
+   Activate your environment and run the API locally, execute the following:
+   ```python 
+   python app.py
+   ``` 
+   Run the cell 
+   [Test PII anonymization on our local running API](notebooks/PII%20Anonymization.ipynb#Test-our-text-on-our-local-running-API). 
+   Make sure you rename the file [sample_env file to .env](powerskill/sample_env) and populate it with the relevant values. Use the
+   variable ```bash URL_LOCAL``` as the URL.
+1) ### Build the docker image 
+   Now build the [docker image](Dockerfile) and upload the image to your container registry  
+   For this step you will need docker running so that we can build and test our inference API locally.
```

Comment:
Inference API can be confusing. Consider being more explicit and call it "Anonymization API" or "Presidio inference API" or something similar.

Reply:
Done.